pkgs/top-level/python-packages.nix

```diff
@@ -19123,6 +19123,29 @@ let
     };
   };
 
+  cmdtest = buildPythonPackage rec {
+    name = "cmdtest-${version}";
+    version = "0.15";
+    disabled = isPy3k || isPyPy;
+
+    buildInputs = with self; [ cliapp ttystatus ];
+    propagatedBuildInputs = with self; [ cliapp ttystatus markdown ];
+
+    # TODO: cmdtest tests must be run before the buildPhase
+    doCheck = false;
+
+    src = pkgs.fetchgit {
```

Comment:
we prefer tarball releases over git, also there is a newer version at `http://code.liw.fi/debian/pool/main/c/cmdtest/cmdtest_0.18.orig.tar.xz`